android/app/src/main/java/com/microsoft/codepush/react/CodePushUpdateUtils.java

```diff
@@ -97,7 +97,9 @@ private static String computeHash(InputStream dataStream) {
     }
 
     public static void copyNecessaryFilesFromCurrentPackage(String diffManifestFilePath, String currentPackageFolderPath, String newPackageFolderPath) throws IOException {
-        FileUtils.copyDirectoryContents(currentPackageFolderPath, newPackageFolderPath);
+        if (currentPackageFolderPath != null) {
```

Comment:
In a condition like this I would also expect checking if the path exists on disk.